images/decode.go

```diff
@@ -27,6 +30,20 @@ func Decode(fileName string) (image.Image, error) {
 	return decodeImageData(fb, file)
 }
 
+func DecodeFromURL(path string) (image.Image, error) {
+	res, err := http.Get(path)
```

Comment:
Consider adding a timeout:
```suggestion
	client := http.Client{
		Timeout: 5 * time.Second,
	}
	res, err := client.Get(path)
```

Reply:
This suggestions feature is cool.